std/random.d

```diff
@@ -527,20 +527,25 @@ alias MinstdRand = LinearCongruentialEngine!(uint, 48271, 0, 2147483647);
 The $(LUCKY Mersenne Twister) generator.
  */
 struct MersenneTwisterEngine(UIntType, size_t w, size_t n, size_t m, size_t r,
-                             UIntType a, size_t u, size_t s,
-                             UIntType b, size_t t,
-                             UIntType c, size_t l)
+                             UIntType a,
+                             uint u, UIntType d,
+                             uint s, UIntType b,
+                             uint t, UIntType c,
+                             uint l)
```

Comment:
OK, cool, thanks!  I'm trying to find ways in which to minimize the diff so as to make it most informative, but TBH I think having fixed-size template parameters is probably preferable.